### DISCUSSION TITLE
config/configuration

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#155] [Announcements]
# config/configuration

Initial config Field Description force The program will preferentially read the configuration from the environment variable, set force to true to force the program to read the c...

https://alist.nn.ci/config/configuration.html

**Comment by ZzeMR** [comment]
Hello,
"By default AList reads the configuration from environment variables, set this field to true to force AList to read config from the configuration file."
Does this mean that i can configure my Alist using env variables inside my docker compose file ?
if so that will be so helpful but i can't find a doc for env var names.
A small table of env variables names would be so helpful !
